llvm/include/llvm/Support/Error.h

```diff
@@ -1420,6 +1420,32 @@ inline Error unwrap(LLVMErrorRef ErrRef) {
       reinterpret_cast<ErrorInfoBase *>(ErrRef)));
 }
 
+/// Common scenarios macros for handling Error or Expect<T> returns
+/// Forward Error.
+#ifndef RETURN_IF_ERROR
```

Comment:
I wonder if it's clearer to add an `LLVM_` prefix to these macros so that in downstream usage it's clear which macro we want for which type (e.g. llvm::Error vs absl::Status). We still want the ifndef I think just to be cautious. What do you think?

Reply:
not too opposed, not sure how pervasive that is - I see stuff like LLVM_VERSION, which makes sense (and other similar cases); this, I'm on the fence. Waiting for others' feedback.